pkg/api/kptfile/v1alpha2/validation.go

```diff
@@ -147,13 +143,50 @@ func validateFnConfigPath(p string) error {
 	return nil
 }
 
+func GetValidatedFnConfigFromPath(pkgPath types.UniquePath, configPath string) (*yaml.RNode, error) {
```

Comment:
Please add comment to describe this function.

Reply:
done